changelogs/fragments/facts_vlan_issue.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - facts fixed for handling l2 devices.
```

Comment:
We can be a bit verbose on what we're fixing and how the behaviour is changing here.